catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -105,6 +119,18 @@ public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
                 "Paimon catalog uri config",
                 false /* immutable */,
                 null /* defaultValue */,
+                false /* hidden */),
+            stringOptionalPropertyEntry(
```

Comment:
Could you make `GRAVITINO_JDBC_USER` and `GRAVITINO_JDBC_PASSWORD` hidden?

Reply:
updated.